script.js

```diff
@@ -1,4 +1,21 @@
 (function() {
+	const paramString = window.location.search.substr(1);
```

Comment:
```suggestion
	const paramString = window.location.search.substr(1); // cut off `?` symbol
```
а то тоже не самая очевидная строчка